src/components/ListItem.js

```diff
@@ -34,14 +35,39 @@ const ListItem = ({ listItem, itemId }) => {
   const isChecked =
     fromMilliSecToHours(getUTCNowInMilliSec() - listItem.recentPurchase) < 24;
 
+  // This function specifies what color each item should have based on how soon the item is to be bought.
+  // also return arial-label each item should have
+  const getStatusAndAriaLabel = ({ howSoon, recentPurchase }) => {
+    if (isOutOfDate(howSoon, recentPurchase)) {
+      return ['inactive', 'inactive item'];
+    }
+
+    return howSoon <= 7
```

Comment:
Maybe having too many logic in the **return** ( concatenating ternary operators ) makes the code a little bit hard to read, an option can be to extract some of that logic, or use a switch statement

Reply:
Sure! We can replace it with a switch statement. :)

Reply:
Switch statements are definitely easier to read!